client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -296,12 +297,9 @@ function PurchaseMetaPrice( { purchase, translate } ) {
 		period = translate( 'month' );
 	}
 
-	return translate( '%(priceText)s %(currencyCode)s {{period}}/ %(period)s{{/period}}', {
-		args: {
-			priceText,
-			currencyCode,
-			period,
-		},
+	// translators: %(priceText)s is the price of the purchase with localized currency (i.e. "C$10"), %(period)s is how long the plan is active (i.e. "year")
+	return translate( '%(priceText)s {{period}}/ %(period)s{{/period}}', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( '%(priceText)s %(currencyCode)s {{period}}(one-time){{/period}}' )` *ES Score: 7*